recipe/meta.yaml

```diff
@@ -1,41 +1,71 @@
+{% set version = "2020a" %}
 
 package:
   name: tzdata
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/t/tzdata/tzdata-{{ version }}.tar.gz
-  sha256: 5e0a01117b3dfc9be27ef727e8f39e355e29bbc788bcd8fe2110871df5f9a935
+  - url: https://data.iana.org/time-zones/releases/tzdata{{ version }}.tar.gz
+    sha256: 547161eca24d344e0b5f96aff6a76b454da295dc14ed4ca50c2355043fb899a2
+  - url: https://data.iana.org/time-zones/releases/tzcode{{ version }}.tar.gz
+    sha256: 7d2af7120ee03df71fbca24031ccaf42404752e639196fe93c79a41b38a6d669
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: generic
+  # The recipe assumes building on Linux, but we don't have "skip" active since
+  # the linter will complain with "`noarch` packages can't have selectors".
+  # skip: True  # [not linux]
+  ignore_run_exports:
+    - libgcc-ng
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
   host:
-    - python
-    - pip
   run:
-    - python
 
 test:
-  imports:
-    - tzdata
+  commands:
+    - test -f "${PREFIX}/share/zoneinfo/leapseconds"
+    - test -f "${PREFIX}/share/zoneinfo/iso3166.tab"
+    - test -f "${PREFIX}/share/zoneinfo/zone1970.tab"
+    - test -f "${PREFIX}/share/zoneinfo/zone.tab"
+    - test -f "${PREFIX}/share/zoneinfo/tzdata.zi"
+    # Make sure we only package zoneinfo and nothing else.
+    - |
+      dirs="$(
+        find "${PREFIX}" -mindepth 1 -maxdepth 2 \
+        \! -path "${PREFIX}/share" \! -path "${PREFIX}/conda-meta*"
+      )"
+      test "${dirs}" = "${PREFIX}/share/zoneinfo"
+    # Make sure we only package timezone information files.
+    - |
+      heads="$(
+        find "${PREFIX}/share/zoneinfo" -type f \
+          \! -name \*.zi \! -name \*.tab \! -name leapseconds \
+          -exec head -c4 {} \; -printf \\n \
+          | uniq
+      )"
+      test "${heads}" = TZif
 
 about:
-  home: https://github.com/python/tzdata
-  license: Apache-2.0
+  home: https://www.iana.org/time-zones
+  dev_url: https://github.com/eggert/tz
+  doc_url: https://data.iana.org/time-zones/tz-link.html
+  license: Public Domain
   license_file: LICENSE
-  summary: Provider of IANA time zone data
+  summary: The Time Zone Database (called tz, tzdb or zoneinfo)
 
   description: |
-    This is a Python package containing zic-compiled binaries for the IANA time zone database.
-    It is intended to be a fallback for systems that do not have system time zone data installed (or don't have it installed in a standard location),
-    as a part of PEP 615.
-  
+    The Time Zone Database (called tz, tzdb or zoneinfo) contains code and
+    data that represent the history of local time for many representative
+    locations around the globe.  It is updated periodically to reflect
+    changes made by political bodies to time zone boundaries, UTC offsets,
+    and daylight-saving rules.
+
 extra:
   recipe-maintainers:
     - ocefpaf
-    - pganssle
```

Comment:
Was this intended?

Reply:
Yep: https://github.com/conda-forge/tzdata-feedstock/pull/1#issuecomment-704409215

Reply:
BTW, I already "forced" @pganssle in the original tzdata, now python-tzdata, @mbargull is doing the polite thing by asking him first since this package was changed in-place.

Reply:
See https://github.com/conda-forge/tzdata-feedstock/pull/1#issuecomment-704409215:
> ..., @pganssle? Would you like to help maintaining this recipe, too? I took you off the maintainers list just to not enforce an additional recipe onto you -- but of course we'd be happy to have you!

Since I moved the recipe that he "signed up for" to https://github.com/conda-forge/python-feedstock, this one is essentially a "new" one. I'd be very happy to have @pganssle as a maintainer for this one too, but did not want to impose any role without asking!

Reply:
> BTW, I already "forced" @pganssle ...

:laughing:. I only now read https://github.com/conda-forge/staged-recipes/pull/11801:
You did write `@pganssle do you want to be a maintainer here?` there -- you did the polite thing too :wink:!
BTW, it's good to see that [@pganssle's comment there](https://github.com/conda-forge/staged-recipes/pull/11801#issuecomment-637569942) also reflects what we're doing here now, which is a nice confirmation!

Reply:
Oh ok cool. Sorry I had not read everything and thought this might be a typo. Feel free to mark resolved.